### ISSUE TITLE
(docs) finalize docs.intelmq.org

### ISSUE THREAD

**Issue by aaronkaplan** [certtools/intelmq#2359]
- [x] replace all links to old docs with the httpS://docs.intelmq.org/... link
- [x] enforce HTTPS
- [ ] monitor it?
- [ ] (maybe) work on the CSS template

Also organise the old trainings etc into https://training.intelmq.org

**Comment by sebix** [comment]
>     * [ ]  enforce HTTPS

The certificate of https://docs.intelmq.org/ is not valid for that domain

![image](https://user-images.githubusercontent.com/199050/236634424-47e9a6a5-a617-4c88-87db-d9b2ad1bfd5a.png)

**Comment by aaronkaplan** [comment]
thx! working on it

**Comment by gethvi** [comment]
In the "Pages" settings of the repo there should be a checkbox to "Enforce HTTPS" (somewhere under the custom domain settings). Checking it will cause GitHub to generate correct certificate and as the name suggests force all connections to HTTPS.

**Comment by aaronkaplan** [comment]
@gethvi yup, I saw that, but that does *not* do the trick. It's a bit more complex but did not have some time to fully debug yet

**Comment by aaronkaplan** [comment]
https is ok now
